build.bat

```diff
@@ -0,0 +1,169 @@
+@echo off
```

Comment:
Sorry, the build process on all of the supported platforms should be done via [cmake](https://cmake.org/). I'm not accepting any custom build scripts, especially the ones that you have to modify for some reason (according to the instructions in README).

Reply:
Unluckly i wasn't able to compile nothing with conan + cmake.
Although i've succeeded in generating the solution file for visual studio the compiled SDL2.lib and SDL2_Mixer.lib from cargo has some dependency that the linker can't figure it out.
Anyway i've updated the conanfile.txt with the dependencies and the readme.md with the instruction for building with windows.
:(

Reply:
@gianmaria Why didn't you file an issue complaining about bad documentation according to [Contribution Rules](https://github.com/tsoding/nothing/blob/master/CONTRIBUTING.md#issues)?